node-module-system/divide.js

```diff
@@ -0,0 +1,2 @@
+const quotient = (a, b) => a / b;
+export default quotient;
```

Comment:
These function names are OK, but it's uncommon to have the function name differ from the file name.